examples/object_detector/index_to_name.json

```diff
@@ -1,14 +1,93 @@
-{"object_type_names" : [
-    "__background__", "person", "bicycle", "car", "motorcycle", "airplane", "bus",
-    "train", "truck", "boat", "traffic light", "fire hydrant", "N/A", "stop sign",
-    "parking meter", "bench", "bird", "cat", "dog", "horse", "sheep", "cow",
-    "elephant", "bear", "zebra", "giraffe", "N/A", "backpack", "umbrella", "N/A", "N/A",
-    "handbag", "tie", "suitcase", "frisbee", "skis", "snowboard", "sports ball",
-    "kite", "baseball bat", "baseball glove", "skateboard", "surfboard", "tennis racket",
-    "bottle", "N/A", "wine glass", "cup", "fork", "knife", "spoon", "bowl",
-    "banana", "apple", "sandwich", "orange", "broccoli", "carrot", "hot dog", "pizza",
-    "donut", "cake", "chair", "couch", "potted plant", "bed", "N/A", "dining table",
-    "N/A", "N/A", "toilet", "N/A", "tv", "laptop", "mouse", "remote", "keyboard", "cell phone",
-    "microwave", "oven", "toaster", "sink", "refrigerator", "N/A", "book",
-    "clock", "vase", "scissors", "teddy bear", "hair drier", "toothbrush"
-]}
\ No newline at end of file
+{
```

Comment:
This would break existing handlers, I don't see a strong reason to do this.

Reply:
The actual handler code can still handle the old case, so it's backwards compatible. This change was to bring consistency between classifiers and object detectors which had no reason as far as I could tell to have a different format for index_to_name.json